src/mercury_engine_data_structures/dread_resource_names.json

```diff
@@ -89438,356 +89438,286 @@
         "versions": []
     },
     "packs/maps/s010_cave/s010_cave.pkg": {
-        "crc": 9219568898208268146,
-        "versions": []
+        "crc": 9219568898208268146
     },
     "packs/maps/s010_cave/subareas": {
         "crc": 12999223075944853031,
         "versions": []
```

Comment:
Seems like you missed a bunch. All empty versions arrays should be removed, right?

Reply:
I could do it but as the opening text says:
> ...and remove all empty versions array for pkg files.

That's not a PKG file. 

I still don't know what was the intended meaning of "no versions array" vs. "empty versions array" since none of my intuitive understanding matches with the data in the file.

Reply:
I read through/debugged the code, and it looks like having the empty versions array just breaks that function, so I think it's best to just update all of them now. Unless we're misunderstanding and @steven11sjf can correct us?